### DISCUSSION TITLE
Accidentally uninstalled/trashed Databases@0.0.0 in Flipper client app - how to get it back?

### DISCUSSION THREAD

**Discussion by theokester** [facebook/flipper#5003] [Q&A]
We have enabled the Database plugin in our Android client application and using the Flipper desktop app on Mac.  One of our testers accidentally disabled the Database plugin in their UI and then opened the Disabled section to re-add it and accidentally clicked the trash can rather than the plus icon.  This caused the Plugin to vanish completely from the list and now we can't figure out how to get it back.  

We tried:
- Reload Flipper from menu (Command R)
- Disconnect and reconnect our Android app to reload it
- Delete the ~/.flipper folder
- Fully delete and reinstall Flipper
- Fully delete and reinstall our Android app

Is there someplace we can download the tar file for the databases@0.0.0 plugin to reinstall it from the package or is there some other way to force reloading it?

**Comment by devphilipcesar** [comment]
Any update on this?

**Comment by ivanarifin** [comment]
any updates?

**Comment by YannickGrenier** [comment]
Had the same issue, fixed it by going into settings and pressing "Reset all state"

<img width="224" alt="image" src="https://github.com/user-attachments/assets/d2ab5d34-0816-4b43-92af-232a666dcb34">
<img width="555" alt="image" src="https://github.com/user-attachments/assets/c8d12869-9b7c-4258-b119-10081550e813">
